### DISCUSSION TITLE
Embeds not working in StackBlitz?

### DISCUSSION THREAD

**Discussion by hnuecke** [vidstack/player#1188] [Q&A]
**Current Behavior:**
Youtube and Vimeo Videos do not pay and do not show Thumbnails an a StackView Example

**Expected Behavior:**
Youtube and Vimeo videos should behave the same as an MP4 Video

**Steps To Reproduce:**
Pick: VUE - VITE - YOUTUBE - DEFAULT THEME
on the INSTALLATION Page. Then go to [examples](https://github.com/vidstack/examples/tree/main/player/vue) and there to
vue/default-theme/README
And click "Open in StackBlitz". 

**Background Indo**
I used the different examples last week; great Idea and VERY helpful! And everything worked like a charme!
After running into a problem in a own demo setup I built with VSCode and your lib, I wanted to check again a similar example to compare with my config; to find the differences.
BUT: The example I opened in StackBlitz does not work neither!
I have no idea what I'm making wrong... ;-(

I created a short video showing my problem. Hopefully you understand my problems and can give me a hint!

Regards
Hans

https://github.com/vidstack/player/assets/3041318/cdd2a176-2117-4ee8-932a-14ac2d0f24ac

**Comment by mihar-22** [comment]
It seems to be working locally but failing in StackBlitz at the moment. Not entirely sure why that is, I'll check again tomorrow.

**Reply by hnuecke** [reply]
StackBlitz still shows the same effect ;-(
**youtube** and **vimeo** videos cannot be started:
controls without effect or, after an .mp4 video was started: no controls for youtube and vimeo at all

Is there any Update available?

**Comment by Curetix** [comment]
Embeds / iframes aren't working in the Stackblitz preview window in general. It works when you open the preview in a separate tab.
